query/using_fhir_views.ipynb

```diff
@@ -0,0 +1,1949 @@
+       "      <td>NaN</td>\n",
+       "    </tr>\n",
+       "  </tbody>\n",
+       "</table>\n",
+       "</div>"
+      ],
+      "text/plain": [
+       "   obs_id patient_id          code_sys                                  code  \\\n",
+       "0   10929      10392  http://loinc.org  1088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "1   11327      10669  http://loinc.org  1088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "2   12026      11149  http://loinc.org  5085AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "3   10063       9040  http://loinc.org  5088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "4   10798      10392  http://loinc.org  5088AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "5   10030       9040  http://loinc.org  159800AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "6   10803      10392  http://loinc.org  159800AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "7   11995      11149  http://loinc.org  159800AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "8   11554      10669  http://loinc.org  1261AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "9   10880      10392  http://loinc.org  1111AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "10  10075       9040  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "11  10083       9040  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "12  10825      10392  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "13  11002      10392  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "14  11310      10669  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "15  11961      11149  http://loinc.org  1271AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "16  11432      10669  http://loinc.org  1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "17  11561      10669  http://loinc.org  1250AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "18  11913      11149  http://loinc.org  1265AAAAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "19  10197       9040  http://loinc.org  159911AAAAAAAAAAAAAAAAAAAAAAAAAAAAAA   \n",
+       "\n",
+       "    val_quantity  \n",
+       "0            NaN  \n",
+       "1            NaN  \n",
+       "2        188.660  \n",
+       "3         36.921  \n",
+       "4         36.833  \n",
+       "5            NaN  \n",
+       "6            NaN  \n",
+       "7            NaN  \n",
+       "8            NaN  \n",
+       "9            NaN  \n",
+       "10           NaN  \n",
+       "11           NaN  \n",
+       "12           NaN  \n",
+       "13           NaN  \n",
+       "14           NaN  \n",
+       "15           NaN  \n",
+       "16           NaN  \n",
+       "17           NaN  \n",
+       "18           NaN  \n",
+       "19           NaN  "
+      ]
+     },
+     "execution_count": 131,
+     "metadata": {},
+     "output_type": "execute_result"
+    }
+   ],
+   "source": [
+    "# TODO: fix the `where` clause Spark SQL issues.\n",
```

Comment:
@omarismail94 the commented out `where` clause below is one example of the Spark SQL issues we discussed off-PR.